### ISSUE TITLE
Vulkan MissingAcquireWait Validation Errors

### ISSUE THREAD

**Issue by sanderPostma** [KhronosGroup/OpenXR-SDK-Source#462]
Running the xr_hello app on Windows 10 using the Vulkan driver (Both Vukan & Vulkan2 settings) I am getting the following error in my console Window every few milliseconds.
```
[10:20:39.791][Error  ] ERROR: (IMAGE 0x967dd1000000000e) Validation Error: [ UNASSIGNED-VkPresentInfoKHR-pImageIndices-MissingAcquireWait ] Object 0: handle = 0x967dd1000000000e, type = VK_OBJECT_TYPE_IMAGE; Object 1: handle = 0xe7e6d0000000000f, type = VK_OBJECT_TYPE_FENCE; | MessageID = 0x1b6b9ef2 | vkQueuePresentKHR(): pPresentInfo->pImageIndices[0] was acquired with a fence VkFence 0xe7e6d0000000000f[] that has not since been waited on
```
I am using the Vulkan SDK version 1.3.275.0
I am wondering if this can be fixed in the hello_xr code, or that it's somewhere more deep. The app is working though, but logging that many error doesn't seem good for performance.

**Comment by sanderPostma** [comment]
Maybe an import detail to mention; I am using this runtime:
MetaXRSimulator\meta_openxr_simulator.json

**Comment by rpavlik** [comment]
I believe this was fixed in the 1.1.36 patch release, out today.

> hello_xr: Fix Windows mirror window acquire, wait, present loop. ([internal MR 3289](https://gitlab.khronos.org/openxr/openxr/merge_requests/3289))
